ca_gazebo/worlds/willowgarage.world

```diff
@@ -1,11 +1,6 @@
 <?xml version="1.0" ?>
```

Comment:
The <gui> tag never worked for me. I'm not sure how to set the pose. I just took out the comments to clean it up.